internal/repository/question_repository.go

```diff
@@ -32,10 +32,10 @@ func NewQuestionDatabaseRepository(db database.ConnPool, builder squirrel.Statem
 	}
 }
 
-func (r *questionDatabaseRepository) BatchInsert(ctx context.Context, questions []*model.Question, formID int64) ([]*model.Question, error) {
+func (r *questionDatabaseRepository) BatchInsert(ctx context.Context, question *model.Question, formID int64) error {
 	tx, err := r.db.Begin(ctx)
 	if err != nil {
-		return nil, fmt.Errorf("session_repository insert failed to begin transaction: %e", err)
+		return fmt.Errorf("session_repository insert failed to begin transaction: %e", err)
```

Comment:
вроде везде v используется
```suggestion
		return fmt.Errorf("session_repository insert failed to begin transaction: %v", err)
```